nemoguardrails/rails/llm/llmrails.py

```diff
@@ -27,15 +27,18 @@
 from nemoguardrails.actions.llm.generation import LLMGenerationActions
 from nemoguardrails.actions.llm.utils import get_colang_history
 from nemoguardrails.context import explain_info_var, streaming_handler_var
+from nemoguardrails.embeddings.cache import CacheEmbeddings
```

Comment:
We should rename this class `EmbeddingsCache`, adhering to the naming convention precedent in `EmbeddingsIndex`

Reply:
Thank you @prasoonvarshney for this suggestion. I will rename it.